docs_src/examples/AppServiceExamples.md

```diff
@@ -1,21 +1,23 @@
+| <div style="width:250px">Example Name</div> | Description |
 | ------------ | ----------- |
-| [Simple Filter XML](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/simple-filter-xml) | Demonstrates Filter of data by device ID and transforming data to XML |
+| [Simple Filter XML](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/simple-filter-xml) | Demonstrates Filtering of Events by Device names and transforming data to XML |
 | [Simple Filter XML HTTP](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/simple-filter-xml-http) |     Same example as #1, but result published to HTTP Endpoint |
 | [Simple Filter XML MQTT](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/simple-filter-xml-mqtt) |     Same example as #1, but result published to MQTT Broker |
-| [Advanced Filter Convert Publish](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/advanced-filter-convert-publish)	| |
-| [Advanced Target Type](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/advanced-target-type)	| |
-| [App Functions AWS](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/aws-export)	| |
-| [App Functions Azure](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/azure-export)	| |
-| [Cloud Event Transforms](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/cloud-event)	| |
-| [Send Command](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/send-command)	| |
-| [Secrets](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/secrets)	| |
-| [Simple CBOR Filter](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/simple-cbor-filter) | |
-
+| [Simple CBOR Filter](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/simple-cbor-filter) | Demonstrates Filtering of Events by Resource names for Event that is CBOR encoded containing a binary reading |
+| [Advanced Filter Convert Publish](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/advanced-filter-convert-publish)	| Demonstrates Filtering of Events by Resource names, custom function to convert the reading and them publish the modified Event back to the MessageBus under a different topic. |
+| [Advanced Target Type](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/advanced-target-type)	| Demonstrates use of custom Target Type and use of HTTP Trigger |
+| [Cloud Export MQTT](https://github.com/edgexfoundry/edgex-examples/tree/master/application-services/custom/cloud-export-mqtt)	| Demonstrates simple custom Cloud transform and exporting to Cloud MQTT Broker. |
```

Comment:
Note this link is dependent on this PR  being merged. https://github.com/edgexfoundry/edgex-examples/pull/66